src/base.jl

```diff
@@ -10,6 +10,8 @@
 @functor Base.Fix2
 @functor Base.Broadcast.BroadcastFunction
 
+@functor Base.Returns
```

Comment:
```suggestion
@static if VERSION >= v"1.7"
  @functor Base.Returns
end
```

Reply:
Thanks! I got distracted, think this is cleaner than adding Compat.jl